_board/feather_m0_express_crickit.md

```diff
@@ -8,9 +8,24 @@ board_url: ""
 board_image: "/assets/images/boards/feather_m0_express_crickit.jpg"
 ---
 
-This board hasn't been fully documented yet. Please make a pull request adding more info to this file.
+**Crickit** is Adafruit's **C**reative **R**obotics & **I**nteractive **C**onstruction **Kit**. It's an add-on to popular Feather ecosystem boards that lets you **#MakeRobotFriend **using CircuitPython.
 
-The description should be written to inform a CircuitPython user what makes the board unique and link to relevant info about it.
+Plug in _any_ Feather mainboard you want into the center, and you're good to go! **The Crickit is powered by seesaw, an I2C-to-whatever bridge firmware. So you only need to use two I2C data pins to control the huge number of inputs and outputs on the Crickit. All those timers, PWMs, sensors are offloaded to the co-processor.**
+
+The only thing that is _not_ managed by seesaw is the audio output. Provided is a small jumper you can solder to connect the audio amplifier to the first analog pin. On Feather M0's this is a true analog output (DAC) and you can play audio clips with CircuitPython. Other Feathers _may not have a DAC!_In that case, you can solder a wire to jumper the audio amp to a PWM pin.
```

Comment:
Add a space before "In":  `_may not have a DAC!_In that`

Reply:
space added!